src/client/commands.ts

```diff
@@ -239,4 +248,4 @@ export function compileLocal(outputChannel: VSC.OutputChannel, diagnosticCollect
             doCompile(Path.join(executableDir, potentialFiles[0]), inputPath, compilerSettings, outputChannel, diagnosticCollection);
         });
     });
-}
\ No newline at end of file
+}
```

Comment:
😞

Reply:
¯\\\_(°╭╮°)_/¯

Reply:
It's all good. I've been forgetting newlines at EOFs, will probably add them to all files in one commit.

Reply:
I hate them :D

Reply:
>I hate them :D

https://stackoverflow.com/a/729795 :D